### ISSUE TITLE
Password recovery finish - token shouldnt be sent in url

### ISSUE THREAD

**Issue by ViliamKopecky** [Einstore/Einstore#130]
`{{SERVER}}auth/recovery/finish?token={{TOKEN}}`

Move it to request body or headers.

Same problem is with `user/invite/finish`

**Comment by rafiki270** [comment]
I thought JS can not process POST?

**Comment by rafiki270** [comment]
Also, is there nay reasoning for this?

**Comment by ViliamKopecky** [comment]
Nope... I'm talking about the request back to API with the new password in JSON. But you acccept the token in URL parameter.

![Screenshot 2019-06-03 at 15 26 21](https://user-images.githubusercontent.com/316435/58805220-f38b3d00-8613-11e9-8b99-c559e1a68d55.png)
